cookbooks/aws-parallelcluster-slurm/templates/default/slurm/login_nodes_daemon_service/login_nodes_on_termination_script.sh.erb

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
```

Comment:
Minor: it's a good practice to put an introductory comment that explains what this script does and howe it should be used

Reply:
Done for daemon and on_termination scripts.